WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/InitializationManager.java

```diff
@@ -73,7 +76,8 @@ private InitializationManager() {
     public static InitializationManager getInstance(String projectName) {
         InitializationManager instance = instances.get(projectName);
         if (instance == null) {
-            instances.put(projectName, new InitializationManager());
+            instance = new InitializationManager();
```

Comment:
Correct. The first call to getInstance() returned null, so I changed the method to ensure it actually returns  an instance.